app/public/humanState.js

```diff
@@ -18,18 +19,40 @@ const HUMAN_STATE = (() => {
             onRoomChange(oldRoom, newRoom);
         }
     }
+    /**
+     * @returns string representation of the room (e.g. 0-0)
+     */
+    const getCurrentRoom = () => {
+        return currentRoom.ROOM_STRING;
+    }
 
     function setRoomChangeCallback(onRoomChangeFunc){
         onRoomChange = onRoomChangeFunc;
     }
 
-    function setCurrentTool(newTool){
-        currentTool = newTool;
-    } 
-
+    function switchHumanTool(tool) {
+        currentTool = tool;
+        // remove all color before color to selected button
+        document.querySelectorAll(".button").forEach((btn) => {
+            btn.classList.remove("selected");
+            btn.classList.add("unselected");
+        });
+        // color to the selected button
+        const currentToolButton = document.getElementById(tool);
+        if (currentToolButton) {
+            currentToolButton.classList.remove("unselected");
+            currentToolButton.classList.add("selected");
+        }
+        CONSOLE.addConsoleLine({
+            message: `Human set tool ${tool}`,
+            style: "public",
+        });
```

Comment:
This console line is being sent locally. Should the server be responsible instead?
The benefit of having the server handle console lines is that it will record all sent lines and can re-send them if either client refreshes their page.
Check out `ConsoleLineData`. Let's send it in `Game` when switching tools.